packages/react/src/transactions/adapters/responders/__tests__/FollowProfilesResponder.spec.ts

```diff
@@ -0,0 +1,137 @@
+import { InMemoryCache } from '@apollo/client';
+import { ProfileFieldsFragment, ProfileFieldsFragmentDoc } from '@lens-protocol/api-bindings';
+import {
+  mockProfileFieldsFragment,
+  createProfileTypePolicy,
+} from '@lens-protocol/api-bindings/mocks';
+import {
+  mockUnconstrainedFollowRequest,
+  mockBroadcastedTransactionData,
+} from '@lens-protocol/domain/mocks';
+
+import { FollowProfilesResponder } from '../FollowProfilesResponder';
+
+function setupTestScenario({ existingProfile }: { existingProfile: ProfileFieldsFragment }) {
+  const apolloCache = new InMemoryCache({
+    addTypename: true,
+    typePolicies: {
+      Profile: createProfileTypePolicy(),
+    },
+  });
```

Comment:
I created the `createMockApolloCache` exactly for this purpose. It gives you an `InMemory` cache with all the type policies and possible types wired in.